### PR TITLE
Changelog for v1.20.0-rc2

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Shouldn't we put changelog entries only for actual release, not release canidates?

**Comment by 6543** [comment]
> Shouldn't we put changelog entries only for actual release, not release canidates?

eg. https://github.com/go-gitea/gitea/releases/tag/v1.19.0-rc0
https://discord.com/channels/322538954119184384/1087684132697550918/1121821586434048083

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,558 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add Link to Stars and Forks Page to Repo List (#24931)
+  * Improve confusable character string (#24911)
+  * Fix install page context, make the install page tests really test (#24858)
+  * Add gitea manager reload-templates command (#24843)
+  * Create pull request for base after editing file, if not enabled on fork (#24841)
+  * Fix video width overflow in markdown, and other changes to match img (#24834)
+  * Support Copy Link for video attachments (#24833)
+  * Improve accessibility when (re-)viewing files (#24817)
+  * Update JS dependencies (#24815)
+  * Bump vm2 from 3.9.17 to 3.9.18 (#24742)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,558 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use globally shared HTMLRender (#24436)
+  * Changelog 1.19.2 (#24365) (#24403)
+  * Fix layouts of admin table / adapt repo / email test (#24370)
+  * Gitea Actions add `base_ref`, `head_ref`, `api_url`, `ref_type` fields (#24356)
+  * Fix 404 error when leaving the last private org team (#24322)
+  * Improve External Wiki in Repo Header (#24304)
+  * Updated upgrade script that is informing user that Gitea service has to be running in order to upgrade it (#24260)
+  * Add run status in action view page (#24223)
+  * Update JS dependencies (#24218)
+  * Bump vm2 from 3.9.15 to 3.9.16 (#24089)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,558 @@ This changelog goes through all the changes that have been made in each release
...
+  * Update github.com/google/go-github to v51 (#23946)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,558 @@ This changelog goes through all the changes that have been made in each release
...
+  * Filter get single commit (#24613)
+  * Make diff view full width again (#24598)
+  * Add permission check for moving issue action in project view page (#24589)
+  * Revert "Prevent a user with a different email from accepting the team invite" (#24531)
+  * Temporarily disable PATs until next release (#24527)
+  * Clean up polluted styles and remove dead CSS code (#24497)
+  * Faster git.GetDivergingCommits (#24482)
+  * Fix test delivery button in repo webhook settings page (#24478)
+  * Use globally shared HTMLRender (#24436)
+  * Changelog 1.19.2 (#24365) (#24403)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

**Comment by wxiaoguang** [comment]
Sorry for bothering and apologize for my mistake .... could it defer after  #25488 and #25484 then release rc2 ? 🙏

**Review by richmahn — APPROVED** [review]

**Comment by richmahn** [comment]
I had to review this changelog for my company since we will be going from 1.17.4 to 1.20.0, so very crucial I know al the changes. This helped to see (along with previous release notes), so that was my review, to make sure it was clear and useful.

**Comment by lunny** [comment]
We should change it to rc2 now.

**Comment by 6543** [comment]
updated :)

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,555 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.20.0-rc1](https://github.com/go-gitea/gitea/releases/tag/1.20.0-rc2) - 2023-06-27
```
Comment (techknowlogick) [inline CHANGELOG.md]:
```suggestion
## [1.20.0-rc2](https://github.com/go-gitea/gitea/releases/tag/1.20.0-rc2) - 2023-06-27
```

**Review by lunny — APPROVED** [review]

**Pull request merged**